src/app/cdk/panel/panels/panels.component.ts

```diff
@@ -57,6 +57,7 @@ export class PanelsComponent implements OnInit {
     label: string
     modal?: ComponentType<any>
     type?: EXTERNAL_ID_TYPE_WORK
+    id?: string
```

Comment:
@Camelia-Orcid why do you added this id here? looks like you are not using it anywhere

Reply:
@amontenegro  https://github.com/ORCID/orcid-angular/pull/1559/files#diff-e6bf5cd98d0dda8e73fa7e5b9924a06df2f53fc27031808b07f2d4971e94f3d6R135

*ngFor="let option of addMenuOptions"
            (click)="add('works', option.action)"
            id="{{ option.id }}"

action and id are attribuites or option object  stored in addMenuOption array